python/py-mediafile/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-mediafile
+version             0.8.0
+revision            0
+
+categories-append   multimedia
+license             MIT
+platforms           darwin
+supported_archs     noarch
+
+description         elegant audio file tagging
+long_description    MediaFile is a simple interface to the metadata tags for many audio file formats. \
+                    It wraps Mutagen, a high-quality library for low-level tag manipulation, with a high-level, \
+                    format-independent interface for a common set of tags.
+
+homepage            http://mediafile.readthedocs.io/
```

Comment:
can you please use `https` to avoid redirects? Other than that looks good and I'll merge!

Reply:
@reneeotten here it is!